### PR TITLE
#92 Map on Where and When page

### PR DESCRIPTION
**Pull request by remrobem**
#92 added map component to the When and Where page with markers for the polling place and voter residence

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/pages/WhereAndWhen.jsx [hunk 1]
```diff
@@ -14,70 +15,100 @@ class WhereAndWhen extends React.Component {
...
+              <b>Polling Place Address: </b>
+              <a
+                target="_blank"
+                href={this._createVoterPrecintMapsLink(pollingPlace.address)}
+                title="View on Google Maps"
+              >
+                {pollingPlace.address}
+              </a>
+            </ListGroupItem>
+            <ListGroupItem>
```
Comment (SchaeStewart) [inline src/components/pages/WhereAndWhen.jsx]: We should hide this `ListGroupItem` when there is no API key present

File: src/components/pages/WhereAndWhen.jsx [hunk 2]
```diff
@@ -14,70 +15,100 @@ class WhereAndWhen extends React.Component {
...
+  _getPollingPlaceInfo = (voter) => {
     if (Object.keys(voter).length === 0) {
-      this.setState({ pollingPlace: {} });
+      this.setState({ pollingPlace: {}, geocode: {} });
       return;
     }
 
     const precinctDesc = voter.precinct_desc;
-    const url = `http://gis.co.guilford.nc.us/arcgis/rest/services/Elections/Elections/MapServer/0/query?where=UPPER(PRECINCT)%20like%20%27%25${precinctDesc}%25%27&outFields=*&outSR=4326&f=json`;
+    // const url = `http://gis.co.guilford.nc.us/arcgis/rest/services/Elections/Elections/MapServer/0/query?where=UPPER(PRECINCT)%20like%20%27%25${precinctDesc}%25%27&outFields=*&outSR=4326&f=json`;
```
Comment (SchaeStewart) [inline src/components/pages/WhereAndWhen.jsx]: Go a head and delete this old url

**Comment by remrobem** [comment]
Ok – I’ll take a look at those changes.

**Comment by SchaeStewart** [comment]
@remrobem Looks good. I'm going to merge it now.

**Pull request merged**